src/tests/Directory.Build.props

```diff
@@ -205,9 +205,9 @@
     <ImportProjectExtensionTargets>false</ImportProjectExtensionTargets>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetOS)' == 'ios' Or '$(TargetOS)' == 'tvos'">
-    <EnableAggressiveTrimming>true</EnableAggressiveTrimming>
-    <PublishTrimmed>true</PublishTrimmed>
+  <PropertyGroup Condition="'$(TargetOS)' == 'ios' or '$(TargetOS)' == 'tvos'">
+    <EnableAggressiveTrimming Condition="'$(OutputType)' == 'Exe'">true</EnableAggressiveTrimming>
```

Comment:
Is OutputType case sensitive?

Reply:
AFAIK MSBuild condition checks are case-insensitive and also from what I can see elsewhere in the repo, we usually check the value of `OutputType` against `'Exe'` or `'Library'`.

Reply:
Thanks for checking out!